packages/zosuss/src/SshBaseHandler.ts

```diff
@@ -15,36 +15,29 @@ import {
     ICommandHandler,
     IOverridePromptConnProps,
     IHandlerParameters,
-    IProfile,
     IHandlerResponseConsoleApi,
     IHandlerFormatOutputApi,
     IHandlerResponseDataApi,
     IHandlerProgressApi,
     IImperativeError,
     ImperativeError,
     ConnectionPropsForSessCfg,
-    SessConstants
+    SessConstants,
 } from "@zowe/imperative";
 import { SshSession } from "./SshSession";
 import { ISshSession } from "./doc/ISshSession";
-
+import { utils } from "ssh2";
+import * as fs from "fs";
 
 /**
  * This class is used by the various handlers in the project as the base class for their implementation.
  */
 export abstract class SshBaseHandler implements ICommandHandler {
-
     /**
      * The session creating from the command line arguments / profile
      */
     protected mSession: SshSession;
 
-    /**
-     * Loaded z/OS SSH profile if needed
-     * @deprecated
-     */
-    protected mSshProfile: IProfile;
```

Comment:
In V2, we can't make the breaking change to remove `mSshProfile` so we should keep it (as deprecated) for now.

Reply:
resolved